### PR TITLE
#1962 - Favicon replacement

### PR DESCRIPTION
**Pull request by ann-aot**
- Added new favicons and logic.
 - Rename the tab title, for now, all the page title is the same, we can customize them in the future if we want as per the router.
- @hellolynn-tbtb  will be testing the below scenarios, after merging,
image

_Screenshots_
Page tile
image
bookmark
image

### CODE REVIEW

File: sources/packages/web/src/router/index.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ const router = createRouter({
 // When tried to return true or route location instead of next with current version of vue-router
 // results in router not working as expected on page refresh.
 router.beforeResolve(async (to, _from, next) => {
+  // For now, all the page title is same, we can customize in future, if we want.
+  document.title = "StudentAid BC";
```
Comment (andrepestana-aot) [inline sources/packages/web/src/router/index.ts:44]: I was wondering if that should come from any constant and be used in BCLogo.vue too for example. Thoughts?

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @ann-aot

**Review by sh16011993 — APPROVED** [review]
👍

File: sources/packages/web/src/router/index.ts (hunk 1 above)
Comment (ann-aot) [reply sources/packages/web/src/router/index.ts:44]: mm, actually I was under the assumption that this is for now and we may be customizing each router title in the future as usual.

@hellolynn-tbtb can you confirm the future plan?

Comment (dheepak-aot) [reply sources/packages/web/src/router/index.ts:44]: I don't have hard feeling to use a constant. I'm ok either way.

**Review by dheepak-aot — APPROVED** [review]
👍

File: sources/packages/web/src/router/index.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/router/index.ts:44]: If there is no intention now to be specific per portal why we cant just set the title in the index.html?
`<title>StudentAid BC</title>`

**Review by andrepestana-aot — APPROVED** [review]
Looks good! I just commented about the constant but that is minor. Either way is ok. Thanks!

**Review by andrewsignori-aot — APPROVED** [review]
👍

File: sources/packages/web/src/router/index.ts (hunk 1 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/router/index.ts]: Although I like it - likely low priority anyway for a specified page title. Let's go with something simple and what we know for now (e.g. StudentAid BC). My two cents. Thanks!

Comment (ann-aot) [reply sources/packages/web/src/router/index.ts]: ok... for now going with `title` tag

**Pull request merged**